src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Intervention.kt

```diff
@@ -0,0 +1,38 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import org.hibernate.annotations.CreationTimestamp
+import java.time.OffsetDateTime
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.validation.constraints.NotNull
+
+enum class AgeGroup {
+  EIGHTEEN_TO_25, OVER_25
+}
+enum class InterventionType {
+  DYNAMIC_FRAMEWORK, STRUCTURED
+}
+
+@Entity
+data class Intervention(
+  @Id @GeneratedValue val id: UUID? = null,
+  @CreationTimestamp val createdAt: OffsetDateTime? = null,
+
+  @NotNull val title: String,
+  @NotNull val description: String,
+
+  @NotNull @Enumerated(EnumType.STRING) var ageGroup: AgeGroup,
```

Comment:
is there a reason for these being `var` not `val`?

Reply:
good spot